src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: 0 auto;
+  width: 726px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    background-color: #eee;
+    height: 100px;
+    width: 100px;
+    border: 1px black solid;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family: Arial, sans-serif;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        font-size: 30px;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: transform 0.5s;
+    }
+  }
+
+}
+
+$days: ("mon": 101px, "tue": 202px, "wed": 303px,
+"thu": 404px, "fri": 505px, "sat": 606px, "sun": 707px);
```

Comment:
Make this variable the numbers from 0 to 6, and calculate the indentation using the formula:

`weekDay * (cellSize + gap)`